apps/src/lab2/views/components/ResizeBar.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import React, {useEffect, useMemo} from 'react';
+import {SeparatorProps} from 'react-resizable-layout';
+
+import moduleStyles from './resizeBar.module.scss';
+
+interface ResizeBarProps {
+  isVertical: boolean;
+  isDragging: boolean;
+  separatorProps: SeparatorProps;
+}
+
+export const RESIZE_BAR_SIZE_PX = 1;
```

Comment:
Not needed?

Reply:
It's still used by `VerticalLayout` and `HorizontalLayout`